src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
 
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  --catalog-columns: 1;
+  gap: 48px;
+  grid-template-columns: repeat(var(--catalog-columns),$card-width);
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    --catalog-columns: 2;
+  }
+
+  @media (min-width: 768px) {
+    --catalog-columns: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --catalog-columns: 4;
+  }
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
```

Comment:
You don't need this rule

Reply:
I delete variable and put stable width 200px